### DISCUSSION TITLE
Parallel Build for Applications Linked to the Same Repository

### DISCUSSION THREAD

**Discussion by jln-dk** [Dokploy/dokploy#877] [Ideas]
Is there a way to enable parallel builds for multiple applications linked to the same repository in Dokploy? Currently, builds seem to run sequentially, which slows down deployments when multiple apps are affected.

Thanks!
Jesper

**Comment by Siumauricio** [comment]
No, currently there is no way, I think a solution could be to add an environment variable and pass it to the bullmq instance since we use it to manage the queues
